pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
I can't ask CloudFlare about their awesome API, because that's a big commercial secret for them :) Very cool - API details are not public info.

```
While we would like to assist you with your inquiry, for the security and privacy of our users we can only work with the account holder directly. If you are the website owner, and account holder, please submit a new support request from the email address you used to register your Cloudflare account. Note that you can also open a ticket directly through the Cloudflare dashboard following the below steps.
```